src/pages/collaborations/collaborationManagementForm.jsx

```diff
@@ -161,9 +211,10 @@ export default function CollaborationManagementForm({
       {success && shouldDisplay && (
         <CustomAlert
           style={{ margin: '0px 24px 20px 24px' }}
-          titleId="COLLABORATION_CREATEDD"
+          titleId="COLLABORATION_CREATED"
           severity="success"
           onClose={() => {
+            queryClient.invalidateQueries(queryKeys.collaborations); // originally put in this in a conditional if(isSuccessful), but got called A LOT
```

Comment:
might be ok to call it a lot.. was it slowing things down?

Reply:
one of the advantages of react-query is that invalidateQueries doesn't immediately trigger a new request, that will only happen if it's not already in progress

Reply:
Done. I suppose that the only time that this will be annoying is when someone is trying to troubleshoot API calls in the network tab... In which case they can just comment this section out.